frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/status-card.tsx

```diff
@@ -87,7 +160,11 @@ export const StatusCard: React.FC<DashboardItemProps> = ({
                 title={t('ceph-storage-plugin~Storage Cluster')}
                 state={cephHealthState.state}
                 details={cephHealthState.message}
-              />
+                popupTitle={isHCPresent ? t('ceph-storage-plugin~Active health checks') : null}
```

Comment:
Active health checks are essentially the ceph warnings/errors that we get from the cephcluster CR that are "active" at that particular moment. I don't think there are inactive health checks because that would mean that a warning healed itself thus it won't be a warning (or a check) anymore.